templates/github/.github/workflows/ci.yml.j2

```diff
@@ -63,31 +63,15 @@ jobs:
     if: github.base_ref == '{{ plugin_default_branch }}'
     needs: "test"
     steps:
-      - name: Fail on deprecations
+      - name: "Download Deprecations"
+        uses: actions/download-artifact@v3
+        with:
+          name: "deprecations"
+      - name: "Print deprecations"
         run: |
-          test -z "{{ "${{ needs.test.outputs.deprecations-pulp }}" }}"
-          {%- if test_azure %}
-          test -z "{{ "${{ needs.test.outputs.deprecations-azure }}" }}"
-          {%- endif %}
-          {%- if test_s3 %}
-          test -z "{{ "${{ needs.test.outputs.deprecations-s3 }}" }}"
-          {%- endif %}
-          {%- if test_lowerbounds %}
-          test -z "{{ "${{ needs.test.outputs.deprecations-lowerbounds }}" }}"
-          {%- endif %}
-      - name: Print deprecations
-        if: failure()
-        run: |
-          echo "{{ "${{ needs.test.outputs.deprecations-pulp }}" }}" | base64 -d
-          {%- if test_azure %}
-          echo "{{ "${{ needs.test.outputs.deprecations-azure }}" }}" | base64 -d
-          {%- endif %}
-          {%- if test_s3 %}
-          echo "{{ "${{ needs.test.outputs.deprecations-s3 }}" }}" | base64 -d
-          {%- endif %}
-          {%- if test_lowerbounds %}
-          echo "{{ "${{ needs.test.outputs.deprecations-lowerbounds }}" }}" | base64 -d
-          {%- endif %}
+          cat deprecations-*.txt | sort -u
```

Comment:
Do we miss "{{ plugin_name }}" in the path?

Reply:
At this point no. As we do no checkout, we don't have the directory "{{ plugin_name }}" to do that. Do you think we should create it for consistency?

Reply:
I would say yes.